e2e/components/PageHeader/PageHeader-test.avt.e2e.js

```diff
@@ -273,18 +231,11 @@ test.describe('PageHeader @avt', () => {
     await expect(page.getByRole('tooltip').getByText('Action 8')).toBeVisible();
     await page.keyboard.press('Tab');
     await expect(page.getByRole('tooltip').getByText('Action 9')).toBeVisible();
-    await page.keyboard.press('Tab');
-    await expect(
-      page.getByRole('tooltip').getByText('Action 10')
-    ).toBeVisible();
```

Comment:
Removed this because it is being collapsed into a menu, which wasn't the case when I opened this PR.

Reply:
Good catch, I opened an issue for this https://github.com/carbon-design-system/ibm-products/issues/5806